.changeset/pre.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
This file will be removed once we publish the next version, no need to take care of it manually.